src/components/subheader/components/description/Description.js

```diff
@@ -0,0 +1,48 @@
+// Libraries
+import PropTypes from 'prop-types'
+
+// components
+import DescriptionBoxList from "../descriptionBoxList";
+import DescriptionAdress from "../descriptionAdress";
+
+// css
+import './description.css'
+
+
+function Description(props){
+
+    const { restaurantInfo, address } = props;
+
+    return (
+        <div className="description inline">
+            <DescriptionAdress address={address} />
+            <DescriptionBoxList restaurantInfo={restaurantInfo}/>
+        </div>
+    )
+}
+
+Description.propTypes = {
+    restaurantInfo : PropTypes.object,
+    address : PropTypes.object
```

Comment:
use PropTypes.shapeOf, because this is a top level check, but what's there inside you should also check